setup.py

```diff
@@ -32,9 +32,10 @@ def run(self):
     'Click>=6.0',
     'docker>=3.5',
     'python-dotenv',
-    'requests',
+    'requests<2.21',
```

Comment:
what is the issue if use higher version?

Reply:
And can we make sure what is the lowest version supported?

Reply:
Similar conflict with docker-compose https://github.com/docker/compose/blob/master/setup.py#L36

Reply:
Lower bound specified.